### PR TITLE
Eg 483 - Create view

### PR DESCRIPTION
**Pull request by woollett-r**
View for 2i review send confirmation

### CODE REVIEW

File: app/controllers/TwoEyeReviewResultController.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  def onSubmit(processId: String): Action[AnyContent] = twoEyeReviewerIdentifierAction.async { implicit request =>
+    val form: Form[TwoEyeReviewResultType] = formProvider()
+
+    form
+      .bindFromRequest()
+      .fold(
+        (formWithErrors: Form[_]) => { Future.successful(BadRequest(view(processId, formWithErrors))) },
+        value => {
+          // Do something with value
```
Comment (cmrhmrc) [inline app/controllers/TwoEyeReviewResultController.scala:62]: ?

Comment (woollett-r) [reply app/controllers/TwoEyeReviewResultController.scala:62]: This task is for the view only so I only have to implement the functionality to display the view.

Comment (cmrhmrc) [reply app/controllers/TwoEyeReviewResultController.scala:62]: ok so no error messaging if they dont select, understood

**Comment by woollett-r** [comment]
Hi Chris,
              There will be errors if the don’t select. The view will get the errors from. formWithErrors.

Regards

**Pull request merged**